packages/cli/core/package.json

```diff
@@ -137,7 +137,7 @@
         "description": "manage Coveo organizations"
       },
       "org:resources": {
-        "description": "manage your organization resources"
+        "description": "manage your organization resources ("
```

Comment:
I don't quite understand this one 🤔

Reply:
that is called a caput volutum keyboardus.